app/helpers/get-relationship.ts

```diff
@@ -0,0 +1,13 @@
+import { helper } from '@ember/component/helper';
+import OsfModel from 'ember-osf-web/models/osf-model';
+
+export function getRelationship([model, relationshipName]: [OsfModel, string]) {
+    if (model === undefined || model === null || relationshipName === undefined || relationshipName === null) {
+        return null;
+    }
+
+    // @ts-ignore
+    return model.belongsTo(relationshipName).id();
+}
+
+export default helper(getRelationship);
```

Comment:
I guess we don't actually need this after all.

Reply:
Should we keep it for the future or just remove the file for now?

Reply:
I tend to prefer to not add things that we don't actually have a current use for. Also, because all of our relationships are by link, there isn't actually a use-case for this.